localization/english/wc_regiment_l_english.yml

```
@@ -14,7 +14,161 @@
  caster_unit_flavor:0 "#F Highly skilled casters, coming from every school of magic, are the backbone of any capable army, as they are able to adapt to any battlefield.#!"
- giant_unit_flavor:0 "#F Feared by many for their sheer size, these massive soldiers tower over any average footman and wipe out large swaths of units with ease.#!"
+ giant_unit_flavor:0 "#F Feared by many for their size, these massive soldiers tower over our average footman and can wipe out large swaths of enemy units with ease.#!"
  aerial_unit_flavor:0 "#F A well armed armada of legendary mounts dominate the skies, leaving those on the ground vulnerable to an attack from any direction.#!"
  massive_aerial_unit_flavor:0 "#F Colossal flying behemoths soar through the sky ravaging all that is below them with terrifying ease.#!"
  riflemen_flavor:0 "#F Warfare changed as firearms became more effective and widely used. Though recognized as relatively inaccurate, especially at longer ranges, the rifles pack a heavy punch and can pierce even the biggest and most heavily armored foes.#!"
+
+###Regional MaA
+ kodo_rider:0 "Kodo Riders"
+ 
+ kodo_rider_flavor:0 "The huge kodos, serving as symbols of might and valor, also use their enormous size and strength to aid our armies in battle."
+ 
+###Cultural MaA
+##Tribal
+ abomination:0 "Abominations"
+ archmage:0 "Archmagi"
+ blademaster:0 "Blademasters"
+ buccaneer:0 "Buccaneers"
+ brute:0 "Brutes"
+ candleholder:0 "Candleholders"
+ couatl:0 "Couatls"
+ dervishes:0 "Dervishes"
+ dire_troll:0 "Dire Trolls"
+ dragon:0 "Dragons"
+ ghoul:0 "Ghouls"
+ gladiator:0 "Gladiators"
+ gryphon_rider:0 "Gryphon Riders"
+ hag:0 "Hags"
+ hexer:0 "Hexers"
+ headhunter:0 "Headhunters"
+ marauder:0 "Marauders"
+ magi:0 "Magi"
+ mountain_giant:0 "Mountain Giants"
+ myrmidon:0 "Myrmidons"
+ savage:0 "Savages"
+ sea_giant:0 "Sea Giants"
+ sentinel:0 "Sentinels"
+ siren:0 "Sirens"
+ snow_hunter:0 "Snow Hunters"
+ swarm:0 "Swarm"
+ thornguard:0 "Thornguards"
+ tidehunter:0 "Tidehunters"
+ ursa:0 "Ursas"
+ warbrave:0 "Warbraves"
+ war_dinosaur:0 "War Dinosaurs"
+ witch_doctor:0 "Witch Doctors"
+ wolfrider:0 "Wolfriders"
+ wyvern_rider:0 "Wind Riders"
+
+ ##Early Medival
+ baradian_marine:0 "Baradian Marines"
+ bearpaws:0 "Bearpaws"
+ blackrock_legionnaire:0 "Legionnaires"
+ chaosborn:0 "Chaosborn"
+ chimaera:0 "Chimaeras"
+ cloud_serpents:0 "Cloud Serpents"
+ faerie_dragon:0 "Faerie Dragons"
+ farstriders:0 "Farstriders"
+ fel_guard:0 "Felguard"
+ dark_ranger:0 "Dark Rangers"
+ dragonhawk_rider:0 "Dragonhawk Riders"
+ golem_arcane:0 "Arcane Golems"
+ golem_mech:0 "Mechanized Golems"
+ huntress:0 "Huntress"
+ peacekeeper:0 "Peacekeepers"
+ proto_drake_rider:0 "Proto Drake Riders"
+ subjagators:0 "Subjugators"
+ watcher:0 "Watchers"
+ wardancer:0 "Wardancers"
+
+ ##High Medival
+ destroyer:0 "Destroyers"
+ frost_wyrm:0 "Frost Wyrms"
+ sapper:0 "Sappers"
+ shredder:0 "Shredders"
+ steam_warrior:0 "Steam Warriors"
+ technomage:0 "Technomages"
+
+ ##Late Medival
```

Minor typo on comment:
```suggestion
 ##Late Medieval
```